### ISSUE TITLE
Clear Thumbnails: Cannot read property 'delete' of null

### ISSUE THREAD

**Issue by shectermen** [avocode/avocode#1466]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.2`
Avocode UID: 46364
```
TypeError: Cannot read property 'delete' of null
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:34:8064
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:6:5958
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3018:46
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1379:16
    at ValueNode.iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1746:12)
    at BitmapIndexedNode.iterate.HashArrayMapNode.iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1739:24)
    at HashArrayMapNode.BitmapIndexedNode.iterate.HashArrayMapNode.iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1739:24)
    at HashArrayMapNode.BitmapIndexedNode.iterate.HashArrayMapNode.iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1739:24)
    at Map.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1377:32)
    at KeyedIterable.mappedSequence.__iterateUncached (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3017:23)
    at seqIterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:606:16)
    at KeyedIterable.Seq.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:276:14)
    at KeyedIterable.forEach (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4383:19)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1224:16
    at Map.withMutations (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1355:7)
    at new Map (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1221:20)
    at reify (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3572:37)
    at Map.map (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4403:14)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:6:5910
    at e.update (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:4:2776)
    at e.updateAll (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:6:5833)
    at Object.n._handleAction [as ID_23] (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:34:8034)
    at Dispatcher._invokeCallback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/flux/lib/Dispatcher.js:198:24)
    at Dispatcher.dispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/flux/lib/Dispatcher.js:174:14)
    at Dispatcher.dispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/flux/lib/dispatcher.js:76:28)
    at t.dispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/flux/lib/action-creator.js:37:24)
    at t.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:20:30141)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:53:4117
    at e._runDataTransaction (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:53:5974)
    at e.clearPreviews (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:53:4033)
    at t._handleClearPreviewsClick (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:58:11884)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:58:9792
    at CallbacksRegistry.apply (/Applications/Avocode.app/Contents/Resources/electron.asar/common/api/callbacks-registry.js:48:42)
    at EventEmitter.<anonymous> (/Applications/Avocode.app/Contents/Resources/electron.asar/renderer/api/remote.js:283:21)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

**Comment by vojtatranta** [comment]
This looks like a bug, what were you trying to do?

**Comment by shectermen** [comment]
Help -> Clear Thumbnail

**Comment by jakubzitny** [comment]
Thanks for the info @shectermen, we'll look into it.

**Comment by shectermen** [comment]
I am glad to help

**Comment by jakubzitny** [comment]
Hi there, we've fixed this bug in latest Avocode `2.14.8`. Please update the app and let us know if you have any more problems or issues 😉
